src/index.js

```diff
@@ -13,41 +13,140 @@ class Task {
 
 const taskArray = [];
 
-const completeTask = (checkbox) => {
-  checkbox.forEach((i) => {
+const editTask = (task, taskContainer) => {
+  const editedTask = document.createElement('input');
+  editedTask.type = 'text';
+  editedTask.className = 'editedTask';
+  editedTask.value = task.textContent;
+  taskContainer.replaceChild(editedTask, task);
+  editedTask.addEventListener('keypress', (e) => {
+    if (e.key === 'Enter') {
+      const taskContainers = document.querySelectorAll('.tasks');
+      const localTasks = JSON.parse(localStorage.getItem('taskList'));
+      for (let i = 0; i < taskContainers.length; i += 1) {
+        if (taskContainers[i].classList.contains('completed-tasks-container')) {
+          localTasks[i].description = editedTask.value;
+          localStorage.setItem('taskList', JSON.stringify(localTasks));
+        }
+      }
+      editedTask.parentElement.classList.remove('completed-tasks-container');
+      taskContainer.replaceChild(task, editedTask);
+      task.textContent = editedTask.value;
+    }
+  });
+};
+
+const removeTask = (task) => {
+  taskContainer.removeChild(task);
+  let counter = 0;
+  const localTasks = JSON.parse(localStorage.getItem('taskList'));
+  const localArray = Array.from(localTasks).filter((i) => i.completed === false);
+  // eslint-disable-next-line array-callback-return, no-multi-assign
+  localArray.map((i) => { i.index = counter += 1; });
+  localStorage.setItem('taskList', JSON.stringify(localArray));
+};
+
+const updateLocal = () => {
+  const localTasks = JSON.parse(localStorage.getItem('taskList'));
+  const tasks = document.querySelectorAll('span');
+  for (let i = 0; i < tasks.length; i += 1) {
+    if (tasks[i].classList.contains('completed-tasks')) {
+      localTasks[i].completed = true;
+    } else {
+      localTasks[i].completed = false;
+    }
+  }
+  localStorage.setItem('taskList', JSON.stringify(localTasks));
+};
```

Comment:
- [ ] When I make a task  done (`completed: true`) and reload the page it becomes again undone `completed: false` it should be saved in the local storage everytime you change the checkbox please try to fix that